Jenkinsfile

```diff
@@ -190,12 +190,12 @@ jobWrapper {
         stage('Aggregate') {
             parallel (
                 cocoa: {
-                    node('docker') {
+                    node('osx') {
                         getArchive()
                         for (cocoaStash in cocoaStashes) {
                             unstash name: cocoaStash
                         }
-                        sh 'tools/build-cocoa.sh'
+                        sh 'tools/build-cocoa.sh -b -x'
```

Comment:
This job specifically isn't supposed to build anything; the builds are done in parallel in the previous steps and then this one just combines them into the release package.

Reply:
I have removed `-b` instead I took the approach of adding the individual arch slices of `lib-realm` & `lib-realm-parser` to the archive that is produced by `cross_compile`